parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -414,4 +414,14 @@ write_files:
   encoding: gzip
   owner: root
   content: !!binary |
-    {{GetVariableProperty "cloudInitData" "customSearchDomainsScript"}}
\ No newline at end of file
+    {{GetVariableProperty "cloudInitData" "customSearchDomainsScript"}}
+
+{{- if (ne GetBootstrapProfileContainerRegistryServer "") }}
+# TODO(binxi): need to update for sovereign cloud.
```

Comment:
do we also need to guard against sov/agc case for now?

Reply:
Currently, this feature is not supported in sov/agc. `GetBootstrapProfileContainerRegistryServer` is empty string in sov/agc, so that this file won't be added.